File: src/mysql_dbs.R
invisible(library(RMySQL))

conn<-dbConnect(MySQL(), user="root", password="hermosillo", host='localhost')

dbs<-dbGetQuery(conn,'show databases')

exclude=c("mysql","sys","performance_schema","information_schema")

dbs<-setdiff(dbs[,1],exclude)

paste0(dbs,collapse=" ")

dbDisconnect(conn)

File: src/mysql_dbs.py
import mysql.connector

try:

	cnx = mysql.connector.connect(user='root', password='hermosillo',host='localhost',port=3306)


	cursor.execute("show databases")


	for db in cursor:
		print db

	cnx.close()

except mysql.connector.Error as err:
	print err
